packages/on-chain-relayer/test/monorepo-integration-tests/scripts/verify-mock-prices-in-adapter.ts

```diff
@@ -19,11 +18,18 @@ import { PriceFeedsAdapterWithoutRounds } from "../../../typechain-types";
     [token: string]: number;
   };
   const bytes32Symbols = Object.keys(pricesToVerify).map(formatBytes32String);
-  const expectedPrices = Object.values(pricesToVerify);
-
   const oracleValues = await contract.getValuesForDataFeeds(bytes32Symbols);
-  expect(oracleValues.length).to.eq(expectedPrices.length);
-  for (let i = 0; i < oracleValues.length; i++) {
-    expect(oracleValues[i]).to.eq(BigNumber.from(expectedPrices[i] * 10 ** 8));
+
+  let oracleValuesIndex = 0;
+  for (const symbol of Object.keys(pricesToVerify)) {
```

Comment:
You have an object you want to iterate through, having access to key, value and index of objects property. More idiomatic way to do this would be

```suggestion
  for (const [oracleValuesIndex, [symbol, price]] of Object.entries(
    pricesToVerify
  ).entries()) {
    const expectedPrice = price * 10 ** 8;
    // ...
  }
```

Reply:
"more idomatic" seems like very opinionated thing, lets focus on important issues :)